docker/docker-compose.dev.yml

```diff
@@ -56,13 +56,10 @@ services:
     environment:
       DATA_DIR: /data
     volumes:
-      - data:/data
+      - ${DOCKER_DATA_DIR}/data:/data
       - ..:/app
     command:
      - /bin/sh
      - -c
      - "pnpm install --frozen-lockfile && pnpm run db:migrate"
 
-volumes:
-  meilisearch:
-  data:
```

Comment:
Why though? Sounds easier to not have to mount a manual data dir?

Reply:
This is mostly up to preference, I would much rather prefer having the /data directory to be persistent so that I can modify the sqlite db in case it is needed